### DISCUSSION TITLE
Automazione per la casa content

### DISCUSSION THREAD

**Discussion by dleone81** [dleone81/apcelettro#45] [General]
Mi serve avere il contenuto della pagina

**Comment by fabiocrazycurls** [comment]
Idem come le altre dobbiamo scriverlo poi creiamo file e lo inseriamo... ma che lunghezza deve avere, di fatto sarà quello che appare una volta cliccato sulla sezione dedicata quindi una frase di effetto con un breve abstract tecnico?

**Comment by dleone81** [comment]
Si corretto

**Comment by dleone81** [comment]
Ciao @Cattalek93 , @fabiocrazycurls  ho rimosso il contenuto dal menu, quando sarà pronto il contenuto lo re-inserirò
